NebulaClient/PacketProcessors/Universe/NameInputProcessor.cs

```diff
@@ -22,18 +22,17 @@ public void ProcessPacket(NameInputPacket packet, NebulaConnection conn)
             }
             using (FactoryManager.EventFromServer.On())
             {
-                // If stellarId > 100 then it must be a planet
-                if (packet.StellarId > 100)
+                if (packet.StarId != -1)
```

Comment:
Maybe introduce a constant field for StarId == -1, would explain what the ID is.